recipe/activate-gcc.sh

```diff
@@ -185,4 +187,21 @@ else
     diff -U 0 -rN /tmp/old-env-$$.txt /tmp/new-env-$$.txt | tail -n +4 | grep "^-.*\|^+.*" | grep -v "CONDA_BACKUP_" | sort
     rm -f /tmp/old-env-$$.txt /tmp/new-env-$$.txt || true
   fi
+
+  # fix prompt for zsh
+  if [[ -n "${ZSH_NAME}" ]]; then
+    _conda_clang_precmd() {
+      HOST="${CONDA_BACKUP_HOST}"
+    }
+
+    [[ -z \$precmd_functions ]] && precmd_functions=()
```

Comment:
Why is there a `\` at the beginning?

Reply:
I don't know. I parts of this from a stack overflow post.

Reply:
fixed!